README.md

```diff
@@ -83,6 +83,74 @@ To create a new version of this package, follow these steps:
 
 [^github-activity]: If you wish, use [`github-activity` to generate a Changelog](https://github.com/choldgraf/github-activity), e.g. `github-activity conda-incubator/conda-store-ui --since 2023.9.1 --until 2023.10.1 --auth <GH personal access token>`
 
+### Running Tests
+
+This repo contains two types of tests: Jest and Playwright
+
+#### Jest tests
+
+To run the Jest tests, run the following command:
+
+```sh
+yarn test
+```
+
+#### Playwright tests
+
+Steps to install and set up:
+
+1. Create Conda environment
+   ```sh
+   conda env create -f environment.yml
+   ```
+2. Activate Conda environment
+   ```sh
+   conda activate conda-store-ui
```

Comment:
Same here, good catch. Hmm... now I'm wondering if it would be simpler for the environment name to match the repo name. If so, I'll change that in this PR. What do you think?